index-Random.js

```diff
@@ -0,0 +1,66 @@
+//Guessing Game Project - Burlington Code Academy
+// started 2.5.20
+// due Monday, Feb. 10 by 9 a.m.
+// Written by Rob Mitchell
+// Version using random guess generator
+
+//set the standard input 
+const readline = require('readline');
+const readlineInterface = readline.createInterface(process.stdin, process.stdout);
+
+// starts game using function
+function guessGame() {
+  let guessCount = 0; //Sets initial count for guesses
+  let maxRange = 100; //Sets initial max range for guessing
+  let minRange = 0; //Sets initial minimum range for guessing
+  //sets up user
+  console.log("Let's play a guessing game where you (human) \nmake up a number and I (computer) try to guess it!")
+  //starts async function
+  async function start() {
+    let rightGuess = null;
+    //asks for an upper bound to the guessing range
+    maxRange = (await ask("Think of a number between zero and...what (this will give me an upper limit)?"))
+    //starts while loop to run while condition is no correct guess
+    while (rightGuess !== "y" || "yes") {
```

Comment:
Oh no! An infinite loop! the string 'yes' is always truthy, so this will loop infinitely. Luckily you're not getting stuck in it because you exit the program on line 35